ucs/core/file_service.py

```diff
@@ -65,7 +65,9 @@ def _assert_update_allowed(
     async def _insert_new(self, file: models.FileMetadataUpsert) -> None:
         """Create a metadata entry for a new file."""
 
-        full_metadata = models.FileMetadata(**file.dict(), latest_upload_id=None)
+        full_metadata = models.FileMetadata(
+            **file.dict(), latest_upload_id=None, object_id=None
```

Comment:
The object ID is only relevant in the context of an upload attempt.

Reply:
Yeah, you're right. The few places the ID was extracted from the metadata object can just have the id supplied as a parameter. See fix in [09eae94](https://github.com/ghga-de/upload-controller-service/pull/45/commits/09eae94e0b6433c962662c63f2b0f3bad65d9b8e) and [f4df54f](https://github.com/ghga-de/upload-controller-service/pull/45/commits/f4df54fc15606333ddb4e33ef0512250d2a04081)